### PR TITLE
[jsk_topic_tools] Ros error for rosparam type conversion

### PR DESCRIPTION
**Pull request by wkentaro**
For: https://github.com/jsk-ros-pkg/jsk_recognition/issues/1188

### CODE REVIEW

File: jsk_topic_tools/src/rosparam_utils.cpp [hunk 1]
```diff
@@ -51,6 +52,7 @@ namespace jsk_topic_tools
     }
     default:
     {
+      JSK_ROS_ERROR("the value cannot be converted into double");
```
Comment (garaemon) [inline jsk_topic_tools/src/rosparam_utils.cpp]: I think `XmlRpc::XmlRpcValue` can be printed by `<<` operator. Can you output it?

Comment (garaemon) [inline jsk_topic_tools/src/rosparam_utils.cpp]: like
```c++
ROS_ERROR_STREAM("the value cannot be converted into double: " << val);
```

**Comment by wkentaro** [comment]
image

I updated the commit.

**Comment by wkentaro** [comment]
```
+ git clone -q https://github.com/ros/catkin

error: RPC failed; result=56, HTTP code = 200

fatal: The remote end hung up unexpectedly

fatal: early EOF

fatal: index-pack failed

++ error
```

**Comment by garaemon** [comment]
may be rejected by github because of too much connection

**Pull request merged**